urbanopt/geojson-modelica-translator
# Enhance and expand microgrid templates and code

#### Any background context you want to provide?
More templates were required to model the components of a micrigrid. This adds them
#### What does this PR accomplish?
New templates and the code to render them for:
- AC inductive loads
- Electrical grids
- Generators
- Batteries
- Capacitors
#### How should this be manually tested?
1. Run the test locally - `poetry run pytest tests/GMT_Lib/test_gmt_lib.py -m "not compilation and not simulation"`
2. Load the resulting models (found in tests/GMT_Lib/output/) in OpenModelica and simulate
3. Expect successful simulations
    1. The `inductive load` models have bad input from UO SDK, so that model is not expected to simulate successfully
#### What are the relevant tickets?
